src/Chirp.Razor/Pages/Public.cshtml.cs

```diff
@@ -15,7 +15,13 @@ public PublicModel(ICheepService service)
 
     public ActionResult OnGet()
     {
-        Cheeps = _service.GetCheeps();
+        var pageString = Request.Query["page"].ToString();
```

Comment:
Can this be done using part instead of Request.Query, and if so would that not make it easier to read?